test/templates/test_discovery-territorial_lord.wgt

```diff
@@ -0,0 +1,59 @@
+[global]
+map="maps/Lesser_Ring.wmf"
+multiplayer="true"
+interactive_player="0"
+
+win_condition="territorial_lord.lua"
+
+# Territorial lord is not time based, but we can use this in the script for the timeout.
+win_condition_duration="3"
+
+# Skip 20 minute delay
+peaceful="true"
+
+fogless="false"
+custom_starting_positions="false"
+
+[player_1]
+tribe="amazons"
+init="discovery.lua"
+team="0"
+ai="empty"
+closed="false"
```

Comment:
team="0" and closed="false" are default values. Should they be included in each file for each player?

Reply:
(see above)